runner.py

```diff
@@ -275,6 +287,25 @@ def process_data(with_indels, dust, evalue, fastq_files, log, max_basecall_itera
     return reference_file
 
 
+def validate_input(output_dir, input_dir, reference_file):
+    if os.path.exists(output_dir):
+        if os.listdir(output_dir):
+            raise Exception("Output_dir must be path to a new or empty directory!")
+    if not os.path.isfile(reference_file):
+        raise Exception("Reference_file must exist!") 
+    if not os.path.isdir(input_dir):
+        raise Exception("Input_dir must exist!")
+    input_dir_loop = os.fsencode(input_dir)
```

Comment:
why fsencode?

Reply:
to loop over the files- make sure there is a fastq file in the folder. 
removed and will be submited as another PR